lms/djangoapps/certificates/tests/test_support_views.py

```diff
@@ -387,11 +352,8 @@ def test_regenerate_cert_with_invalidated_record(self):
         assert response.status_code == 200
         self.assertCertInvalidationExists()
 
-        # Check that the user's certificate was updated
-        # Since the student hasn't actually passed the course,
-        # we'd expect that the certificate status will be "notpassing"
         self.assertGeneratedCertExists(
-            user=self.student, status=CertificateStatuses.notpassing
+            user=self.student, status=CertificateStatuses.downloadable
```

Comment:
This caught my eye. Shouldn't the status be `unavailable` here? If there is still an active CertificateInvalidation record tied to this certificate and we attempt regeneration I would think that the status of the certificate would still be `unavailable`?

Reply:
Ah, this is now a confusing test due to the mock annotation:     _@mock.patch(CAN_GENERATE_METHOD, mock.Mock(return_value=True))_

I think this case is adequately tested in test_generation_handler so I'll just remove it.

Reply:
Ah! Thanks, I get what I was missing now. I agree that it could be confusing (if you miss that annotation as I did). I think it's fine to remove this particular test.